apps/api/src/app/integrations/usecases/update-integration/update-integration.usecase.ts

```diff
@@ -59,7 +77,19 @@ export class UpdateIntegration {
 
     const updatePayload: Partial<IntegrationEntity> = {};
 
-    if (command.active || command.active === false) {
+    if (command.name) {
+      updatePayload.name = command.name;
+    }
+
+    if (command.identifier) {
```

Comment:
```suggestion
    if (identifierHasChanged) {
```
I'd reuse it as we don't need to update the field if it is the same.